tests/openvino/test_decoder.py

```diff
@@ -131,6 +131,12 @@ class OVModelForCausalLMIntegrationTest(unittest.TestCase):
     if is_transformers_version(">=", "4.55.0"):
         SUPPORTED_ARCHITECTURES += ("gpt_oss", "gpt_oss_mxfp4")
 
+    if is_transformers_version(">=", "4.56.0"):
+        SUPPORTED_ARCHITECTURES += ("hunyuan_v1_dense",)
+
+    if is_transformers_version("<", "4.56.0"):
+        SUPPORTED_ARCHITECTURES += ("qwen", "chatglm", "chatglm4")
+
```

Comment:
why it is needed. It looks to be fixed during transition to latest transformers

Reply:
its just copied from legacy version, and let me remove it.